### DISCUSSION TITLE
Seperate the proposed submission-type, from the accepted-as submission-type

### DISCUSSION THREAD

**Discussion by oxinabox** [pretalx/pretalx#1076] [Ideas]
## Problem you are facing
I wish to know what type a submission was proposed by the author,
vs what type of submission the program committee accepted the program item as.

## Possible Solution
Introduce this idea of a a "Accepted-As" submission type, that is set by the program committee, as a distinct field from the submission type as set by the submitter.

## Context

We have 3 main types of submission at JuliaCon.
 - Talk: which is 30 minutes
 - Lightning Talk: which is 10 minutes
 - Poster: which does not include a oral component but does mean you can bring a poster and talk to people in the the poster session.

We always have far more submissions for each category than we can possibly accept.
 
Often, thus the program committee will look at a Talk submission and its reviews, and conclude "We can't offer this presentation a full 30 min talk, but seems like they could get the best parts of it across in a 10 minute Lighting Talk", and so we will change the the submission type to be Lightning Talk then Accept it. (and we make sure our email say "Your submission has been accepted as a Lightning Talk").

Then later it happens that other items that we accepted withdraw, or via some other way we find more space, and we want to go back and reconsider these and give some a second round offer of a full length talk after all.
However, at this point we can't tell anymore which of the Lightning Talks were originally proposed as  Talks (and thus might be interested), and which were originally proposed as Lightning Talks (and thus would not).

**Comment by rixx** [comment]
Thank you for your feedback and the idea! Even though this is not a feature we'll be able to prioritise at the moment, I'll happily help guide a contribution for this feature if anybody is interested in working on this.
